src/outputs/form_output_prep.py

```diff
@@ -52,6 +52,10 @@ def form_output_prep(
         ni_full_responses["602"] = 100
         ni_full_responses["formtype"] = "0003"
 
+        # Rename "turnover" to "frozenturnover"
+        ni_full_responses.rename(
+            columns={"turnover": "frozenturnover"}, inplace=True)
```

Comment:
I think this needs to be removed, I think it's unecessary and would cause an error as I don't think turnover would be in the df now...

Reply:
It doesn't error but pretty sure not needed

Reply:
Yes, removed